docs/content/commands/npm-team.md

```diff
@@ -24,31 +22,76 @@ Used to manage teams in organizations, and change team memberships. Does not
 handle permissions for packages.
 
 Teams must always be fully qualified with the organization/scope they belong to
-when operating on them, separated by a colon (`:`). That is, if you have a `wombats` team in a `wisdom` organization, you must always refer to that team as `wisdom:wombats` in these commands.
+when operating on them, separated by a colon (`:`). That is, if you have a
+`newteam` team in an `org` organization, you must always refer to that team
+as `org:newteam` in these commands.
```

Comment:
does the org require the `@` prefix here? docs below show `@org:newteam` so we should be consistent